plugins/modules/database/saphana/hana_query.py

```diff
@@ -161,7 +181,10 @@ def main():
     if database is not None:
         command.extend(['-d', database])
     # -x Suppresses additional output, such as the number of selected rows in a result set.
-    command.extend(['-x', '-i', instance, '-u', user, '-p', password])
+    if userstore:
+        command.extend(['-x', '-U', user])
+    else:
+        command.extend(['-x', '-i', instance, '-u', user, '-p', password])
```

Comment:
BTW, since this is a common security problem: is there a way of providing the password in another way, say via stdin? That would be preferable over passing them on the command line, since that allows other users on the same system to extract the password from `/proc`.

Reply:
for this case is the userstore as an alternative to prevent the password is available anywhere